src/app/modules/login/login-form.layout.ts

```diff
@@ -0,0 +1,25 @@
+import { DynamicFormLayout } from '@ng-dynamic-forms/core';
+
+export const LOGIN_FORM_LAYOUT: DynamicFormLayout = {
+  personalInformation: {
+    element: {
+      control: 'row',
+    },
+  },
+  password: {
+    element: {
+      label: 'required',
+    },
+    grid: {
+      host: 'col-24',
+    },
+  },
+  userName: {
+    element: {
+      label: 'required',
+    },
+    grid: {
+      host: 'col-24',
+    },
+  },
```

Comment:
This is redundant if your form has no inputs on the same row. It will be in single column by default.

Reply:
done